File: 01-FUNDAMENTOS_DE_PROGRAMACAO/Java/Exercicio7.java
/*
7 - Elabore um programa para calcular e escrever a seguinte soma:
    37 * 38       36 * 37     35 * 36             1 * 2
S = ---------- + --------- + ---------- + ..... + ------
        1           2           3                   37
 
*/
public class exercicio7 {
    public static void main(String[] args) {
        int numerador1=37, numerador2=38, denominador;
        float soma=0;
        
        for(denominador=1; denominador<=37; denominador++){
            soma += ((numerador1*numerador2)/(float)denominador);
            numerador1--;
            numerador2--;
        }
        
        System.out.println("Resultado da somatória: "+soma);
    }
}

File: 01-FUNDAMENTOS_DE_PROGRAMACAO/Dart/exercicio7.dart
// 7 - Elabore um programa para calcular e escrever a seguinte soma:
//      37 * 38      36 * 37     35 * 36             1 * 2
// S = ---------- + --------- + ---------- + ..... + ------
//         1           2            3                   37

main(List<String> args) {
  int denominadorA = 37;
  int denominadorB = 38;
  double soma = 0;
  for (int i = 1; i <= 37; i++) {
    soma += ((denominadorA * denominadorB) / i);
    denominadorA--;
    denominadorB--;
  }
  print('A somatória é $soma');
}
